requirements.txt

```diff
@@ -5,12 +5,14 @@ click==6.6
 cryptography==1.4
 fauxfactory==2.0.9
 Inflector==2.0.11
+import_string==0.1.0
```

Comment:
do we need to install it? AFAIK it is dependency of other module `manage` and manage installs its itself.